cpp/CMakeLists.txt

```diff
@@ -384,7 +384,8 @@ add_library(cudf
     src/transform/mask_to_bools.cu
     src/transform/nans_to_nulls.cu
     src/transform/transform.cpp
-    src/transpose/transpose.cu
+    src/transform/row_bit_count.cu
```

Comment:
I know this is minor but could you put `transform/row_bit_count.cu` alphabetically in front of `transform/transform.cpp` here?